src/hotspot/share/runtime/flags/jvmFlag.hpp

```diff
@@ -115,7 +114,6 @@ class JVMFlag {
     DIAGNOSTIC_FLAG_BUT_LOCKED,
     EXPERIMENTAL_FLAG_BUT_LOCKED,
     DEVELOPER_FLAG_BUT_PRODUCT_BUILD,
-    NOTPRODUCT_FLAG_BUT_PRODUCT_BUILD
```

Comment:
Should the ',' on the previous line be removed?

Reply:
Yes, I guess our compilers don't complain about that anymore.